block-util/src/config/mod.rs

```diff
@@ -66,15 +62,43 @@ pub trait BlockchainConfigExt {
 }
 
 impl BlockchainConfigExt for BlockchainConfig {
-    fn validate_params(
-        &self,
-        _relax_par0: bool,
-        _mandatory_params: Option<Dict<u32, ()>>,
-    ) -> Result<bool> {
-        // TODO: refer to https://github.com/everx-labs/ever-block/blob/master/src/config_params.rs#L452
-        // STUB: currently should not be invoked in prototype
-        // todo!()
-        Ok(true)
+    fn validate_params(&self) -> bool {
+        match self.get::<ConfigParam0>() {
```

Comment:
It can be greatly simplified with `let .. else`.

And it might be useful to return `Result<()>` instead of just bool to know what is wrong with the config.